src/Orleans.Runtime/Placement/ResourceOptimizedPlacementDirector.cs

```diff
@@ -71,23 +69,11 @@ public Task<SiloAddress> OnAddActivation(PlacementStrategy strategy, PlacementTa
             return Task.FromResult(compatibleSilos[Random.Shared.Next(compatibleSilos.Length)]);
```

Comment:
Minor and not from this PR, but why not returning `compatibleSilos[0]`?

Reply:
I want it to degrade to random. I think you missed part of the code since GH hid it:
![image](https://github.com/dotnet/orleans/assets/203839/fc029a46-bc08-438c-a1e2-8e599140344f)